iocage/lib/iocage.py

```diff
@@ -972,11 +1013,13 @@ def set(self, prop, plugin=False):
         if "template=no" in prop:
             self.jail = f"{self.jail} (template)"
 
+        key, value = prop.split("=", 1)
```

Comment:
@skarekrow the second parameter of the split function is interesting for other parts of iocage as well.

```
>>> "foo=bar=baz".split("=")
['foo', 'bar', 'baz']
>>> "foo=bar=baz".split("=", 1)
['foo', 'bar=baz']
```